app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1><%= @recipe.name %></h1>
+
+<table>
+  <tr>
+    <td>Name: <%= @recipe.name %></td>
+  </tr>
+  <tr>
+    <td>Complexity: <%= @recipe.complexity %></td>
+  </tr>
+  <tr>
+    <td>Genre: <%= @recipe.genre %></td>
+  </tr>
+  <tr>
+    <td>Total cost: <%= @cost %></td>
+  </tr>
+</table>
```

Comment:
You're doing awesome with the html table syntax. Keep in mind as an alternative that it's totally fine to do an unordered list if that feels easier! But it seems like you've got the table down pat :)